### ISSUE TITLE
Don't crash if there are no writable databases available

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-upload-csvs#30]
This can happen when running plugins like this one:

- https://github.com/simonw/datasette-upload-dbs/issues/5

**Comment by simonw** [comment]
From that bug report:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/datasette/app.py", line 1254, in route_path
    response = await view(request, send)
  File "/usr/local/lib/python3.9/dist-packages/datasette/app.py", line 1442, in async_view_fn
    response = await async_call_with_supported_arguments(
  File "/usr/local/lib/python3.9/dist-packages/datasette/utils/__init__.py", line 977, in async_call_with_supported_arguments
    return await fn(*call_with)
  File "/usr/local/lib/python3.9/dist-packages/datasette_upload_csvs/__init__.py", line 46, in upload_csvs
    db = [
IndexError: list index out of range
INFO:     127.0.0.1:46308 - "GET /-/upload-csvs HTTP/1.1" 500 Internal Server Error
```

**Comment by simonw** [comment]
Related:
- #28

**Comment by simonw** [comment]
May as well solve both of these at once.

**Comment by simonw** [comment]
I'm going to solve this one as a bug fix, rather than bundling the fix into the work on #28.

**Comment by simonw** [comment]
Refs:
- https://github.com/simonw/datasette/issues/1807
